.make.defaults

```diff
@@ -219,3 +219,56 @@ __check_defined = \
 	pip install wheel;			\
 	pip install -r requirements.txt;	\
 
+.PHONY: .defaults.check.installed
+.defaults.check.installed::
+	@installed=$$(which $(CHECK_RUNNABLE));		\
+	if [ -z "$$installed" ]; then			\
+	    echo $(CHECK_RUNNABLE) must be installed;	\
+	    exit 1;					\
+	fi
+
+.PHONY: .defaults.minio.check
+.defaults.minio.check::
+	@# Help: Check that minio and the mc CLI is installed. 
+	$(MAKE) CHECK_RUNNABLE=minio .defaults.check.installed
+	$(MAKE) CHECK_RUNNABLE=mc .defaults.check.installed
+
+MINIO_DIR=/tmp/data-prep-lab
+MINIO_ALIAS=local
+# These are the credentials used by samples.
+MINIO_ADMIN_USER=localminioaccesskey
+MINIO_ADMIN_PWD= localminiosecretkey
+
+.PHONY: .defaults.minio.stop
+.defaults.minio.stop::
+	$(MAKE) .defaults.minio.check
+	@# Help: Stop the minio server used for S3-based samples
+	-mc admin service stop $(MINIO_ALIAS) > /dev/null 2>&1
+	-mc alias rm $(MINIO_ALIAS)  > /dev/null 2>&1
+
+.PHONY: .defaults.minio.start
+.defaults.minio.start::
+	$(MAKE) .defaults.minio.stop
+	@# Help: Stop, then start, the minio server used for S3-based samples
+	rm -rf $(MINIO_DIR) 
+	mkdir  $(MINIO_DIR) 
+	echo Starting minio server			
+	minio server $(MINIO_DIR) &
+	sleep 1
+	mc alias set $(MINIO_ALIAS) http://127.0.0.1:9000 minioadmin minioadmin
+	mc admin user svcacct add --access-key "$(MINIO_ADMIN_USER)" --secret-key "$(MINIO_ADMIN_PWD)" $(MINIO_ALIAS) minioadmin
+	mc alias ls $(MINIO_ALIAS)
+	@echo "Test minio server started" 
+
```

Comment:
We have Minio strat and Minio load, where load does start. I would suggest to remove load as an explicit target and add load functionality to start. Less typing